### ISSUE TITLE
guide does not mention which format the USB stick should be in

### ISSUE THREAD

**Issue by Panthium** [osy86/HaC-Mini#290]
I followed the guide to the letter.
my USB stick format type was pre-selected so i didn't change it. 
It turned out my default format type was the wrong type, so my UBS stick doesn't boot after the patch tool was used.
Looking at another Catalina install guide, additional information was provided, which was to make sure to format the USB stick using "Mac OS Extended (Journaled)"otherwise the 2nd boot partition is not created.

please can you add this additional information to the guide ?

Name: Installer
Format: Mac OS Extended (Journaled)
Scheme: GUID Partition Map

**Comment by osy86** [comment]
It says this in the guide

> Give the name Installer and make sure to select GUID Partition Map as the Scheme.

**Comment by Panthium** [comment]
Thanks for responding !
but the problem I had was my hackintosh was not defaulting to format the stick in Mac OS extended (Journaled) but something else. I reinstall my old hackintosh so it’s selecting this Format as the default again.
It was this guide the told me the format was incorrect. The guide only mentions the name: Installer and the Sceme: GUID Partition Map, but forgets to mention the Format:


Now my problem is I get up 12 minutes to go and it reboots.
I’m using Catalina 15.5 image with patcher 2.10
I’ve reset boot args as suggest in 2.10 docs. I also tried  the 2.9 patcher, but  it was worse.

Any ideas ?
I did try the ticking the NVMe fixer as I have a Samsung OEM 1TB NVMe SSD. Tomorrow I will try a different SSD.
